docs/developer-docs/developer-welcome.md

```diff
@@ -0,0 +1,181 @@
+
+When considering whether to vote to approve a revision proposal, please familiarize yourself with the revision goal criteria outlined in the discussion forum pinned post. The goal of a revision is to turn a lacking set into a high-quality, modern set that ideally will never need another revision. Voting in revision proposals is an important responsibility that all developers should take seriously and participate in for the long-term health of the RetroAchievements project.
+
+When proposing a revision, it requires contacting all active authors of the relevant set. If you wish to Opt-Out of being contacted, use [this form](https://forms.gle/mgzv7RHbJEPCrxc77).
+
+For details on how to propose a revision, see [Achievement Set Revisions](/guidelines/content/achievement-set-revisions)
+
+If you are re-authoring the set (keeping the old achievements in Unofficial and promoting your new reworked achievements), make sure to contact DevCompliance for approval. If you want to transfer the earners for some or all of the previous achievements, you will need to reach out to RAdmin or #admin-site-cleanup on Discord for assistance.
+
+The request would include the Game and a full list of achievement transfers in Old ID -> New ID format
+
+Example: 
+
+https://retroachievements.org/game/990
+
+Transfer earners for Game 990:
+
+11702 -> 175183
+
+11709 -> 175184
+
+And so on...
+
+## Emulator Updates
+
+You will now have a few new privileges in the RA emulators to take advantage of.
+
+### Creating Game Pages and Linking Hashes
+
+The biggest new privilege is that you can now link new hashes to games by loading the game in an RA emulator and linking it.
+
+If creating a new page, select "New Title" and write an appropriate name in the text box. Check if there are any junk pages labeled ~Z~ which can be recycled. 
+
+::: info
+Further reading:
+
+[Creating Entries](/guidelines/developers/creating-entries)
+
+[Achievements for ROM Hacks](/guidelines/content/achievements-for-rom-hacks)
+
+[Subsets](/guidelines/content/subsets)
+
+[Working with the Right ROM](/guidelines/content/working-with-the-right-rom)
+:::
+
+![Creating a game page with an unidentified hash](/public/game-page-creation.png)
+
+### Code Notes
+
+You are now able to overwrite and delete any existing code notes. Be careful not to overwrite other’s notes without reason, if you are making improvements or correcting a note that is an acceptable reason.
+
+### Achievements
+
+Developers can directly make edits to achievements in Core.
+
+You can now freely promote and demote any achievements from Unofficial and Core through the emulator but it is much easier to do this through the site.

```

Comment:
Consider adding a note that they no longer have to demote to unofficial to make changes and can publish changes directly in core.  Sometimes this is a new-dev confusion.